<launch>
	<!-- Launch Gazebo Simulation -->
	<arg name="x" default="0.0"/>
    <arg name="y" default="0.0"/>
    <arg name="z" default="0.05"/>
	<arg name="world" default="$(find auto_landing)/worlds/chess_board.world"/>
	<arg name="sdf" default="$(find auto_landing)/models/p450_monocular/p450_monocular.sdf"/>
	<arg name="model" default="p450_monocular"/>

	<!-- <node name="pad_car_controller" pkg="auto_landing" type="pad_car_controller.py" output="screen" /> -->

    <include file="$(find auto_landing)/launch/iris_px4.launch">
	  <arg name="world" value="$(arg world)"/>
	  <arg name="sdf" value="$(arg sdf)"/>
	  <arg name="model" value="$(arg model)"/>
      <arg name="x" value="$(arg x)"/>
      <arg name="y" value="$(arg y)"/>
      <arg name="z" value="$(arg z)"/>
	</include>

	<!-- run the landpad_det.cpp -->
    <!-- <node pkg="prometheus_detection" type="my_landpad_det" name="landpad_det" output="screen">
        <param name="camera_topic" type="string" value="/prometheus/sensor/monocular_down/image_raw" />
        <param name="camera_info" type="string" value="$(find prometheus_gazebo)/config/camera_config/camera_param_gazebo_monocular.yaml" />
    </node> -->



</launch>
